ReleaseChecklist.md

```diff
@@ -42,17 +52,18 @@
  - [ ] Update the version and the hash (``sha256sum solidity_$VERSION.tar.gz``) in https://github.com/ethereum/homebrew-ethereum/blob/master/solidity.rb
```

Comment:
```suggestion
 - [ ] Update the version and the hash (``sha256sum solidity_$VERSION.tar.gz``) in [homebrew-ethereum formula](https://github.com/ethereum/homebrew-ethereum/blob/master/solidity.rb)
```

Reply:
Would also suggest a similar change to the line before.

Reply:
Done.